src/base/validator.ts

```diff
@@ -31,6 +36,26 @@ export function validateTimeValue(value: string) {
   return false;
 }
 
+export function validateMaxMinValue(max: string, min: string) {
```

Comment:
I think this function unnecessary , we can use isFirstTimeEarlier for do that, If you have special reason , you can keep that, if not , please remove

Reply:
@nmanhit 
Because I searched the whole src code and found that functions of "/datetime/utils" are not used out of base except in test code.
So I wrote a function to wrap "isFirstTimeEarlier" and used it in TimePicker.

If we can use functions of "/datetime/utils" in TimePicker, I think we can remove "validateMaxMinValue".
Is it OK?

Reply:
are  you created the isFirstTimeEarlier function ?
If right, i think you can edit this function for easy to use in other case and you should use it
i suggestions you can change this function to

isFirstTimeEarlier >> timeCompare(start_time, end_time)
```
export const timeCompare = (startTime: string, endTime: string) => {
  const startTimeMinutes = convertTimeValueToMinutes(startTime);
  const endTimeMinutes = convertTimeValueToMinutes(endTime);
  if (startTimeMinutes > endTimeMinutes) return 1; // startTime is greater than endTime
  if (startTimeMinutes === endTimeMinutes) return 0; // Both times are equal
  return -1; // startTime is less than endTime
};
```

Reply:
Your suggestion is great.
I have change the function "isFirstTimeEarlier" to "timeCompare" in  "/datetime/utils".
Please check the code here https://github.com/kintone-labs/kintone-ui-component/pull/953/commits/4217819df4acad9a68bbb784300167a3ad151bfb

About "validateMaxMinValue" in validator.ts, do we need to remove it now?

Reply:
>> About "validateMaxMinValue" in validator.ts, do we need to remove it now?
If there is no place uses it, please remove it, and change function in unit test to timeCompare